lib/src/editor/editor_component/service/shortcuts/character_shortcut_events/format_by_wrapping_with_single_char/handle_format_by_wrapping_with_single_char.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:appflowy_editor/appflowy_editor.dart';
+
+enum FormatStyleByWrappingWithSingleChar {
+  code,
+  italic,
+  strikethrough,
+}
+
+Future<bool> Function(EditorState) handleFormatByWrappingWithSingleChar({
+  required String char,
+  required FormatStyleByWrappingWithSingleChar formatStyle,
+}) {
```

Comment:
This function looks a little tricky and confusing. I think you can simplify it by just using editorState as the parameter.